libraries/standard/utilities/include/iot_logging_setup.h

```diff
@@ -114,7 +114,7 @@
  */
 
 /**
- * @def IotLog( messageLevel, pLogConfig, ... )
+ * @def IotLog( messageLevel, pLibraryName, pFormat, ... )
```

Comment:
Given the outcome, this change needs to be reverted?

Reply:
Yes, reverted